conf/nginx.conf

```diff
@@ -28,6 +28,15 @@ http {
     listen <%= ENV["PORT"] %>;
     server_name localhost;
 
+    <% if ENV["NEW_RELIC_LICENSE_KEY"] %>
```

Comment:
this line will have to be added to any application using this buildpack wanting to be monitored by newrelic